cl/disclosures/urls.py

```diff
@@ -1,15 +1,15 @@
 from django.urls import path
 
 from cl.disclosures.views import (
-    financial_disclosures_for_somebody,
     financial_disclosures_home,
+    financial_disclosures_viewer,
 )
 
 urlpatterns = [
     path(
-        "person/<int:pk>/<blank-slug:slug>/financial-disclosures/",
-        financial_disclosures_for_somebody,
-        name="financial_disclosures_for_somebody",
+        "financial-disclosures/<int:pk>/<blank-slug:slug>/",
+        financial_disclosures_viewer,
+        name="financial_disclosures_viewer",
```

Comment:
Boy, I guess I didn't realize this would have the person's PK instead of the disclosure's PK. Now that I realize it has to be the person's PK, I feel like flipping back to the old URL pattern. I think. It feels right to me that disclosures be nested inside people, particularly if they're going to have the same ID in the URL. Hm.

Reply:
we use the PK on the end - its jus tread as a parameter

Reply:
it doesn't haaaaave to be the person PK - we could easily load up the slug with the disclosure PK and then call for all related disclosures to populate the page.  Its a little more awkward ... but not by much.  Either way I await your thoughts.